### PR TITLE
Np 1979 use latest nva commons

### PR DESCRIPTION
**Pull request by axthosarouris**

### CODE REVIEW

File: modify-publication/src/test/java/no/unit/nva/publication/modify/ModifyPublicationHandlerTest.java [hunk 1]
```diff
@@ -129,21 +127,6 @@ public void handlerReturnsOKResponseOnValidInput() throws IOException, ApiGatewa
         assertThat(getGatewayResponseBodyStatus(gatewayResponse), is(equalTo(expectedStatus)));
     }
 
-    @Test
```
Comment (axthosarouris) [inline modify-publication/src/test/java/no/unit/nva/publication/modify/ModifyPublicationHandlerTest.java:132]: This test was moved to line 177

File: modify-publication/src/test/java/no/unit/nva/publication/modify/ModifyPublicationHandlerTest.java [hunk 2]
```diff
@@ -181,16 +164,31 @@ public void handlerReturnsInternalServerErrorResponseOnUnexpectedException()
...
         publicationServiceThrowsException();
         modifyPublicationHandler.handleRequest(generateInputStreamWithValidBodyAndHeadersAndPathParameters(
-                        publication.getIdentifier()), output, context);
+            publication.getIdentifier()), output, context);
         GatewayResponse<Problem> gatewayResponse = toGatewayResponseProblem();
         assertEquals(SC_INTERNAL_SERVER_ERROR, gatewayResponse.getStatusCode());
         assertThat(appender.getMessages(), containsString(SOME_MESSAGE));
     }
 
+    @Test
```
Comment (axthosarouris) [inline modify-publication/src/test/java/no/unit/nva/publication/modify/ModifyPublicationHandlerTest.java:177]: Moved from line 132

File: publication-commons/src/main/java/no/unit/nva/publication/exception/InvalidPublicationException.java [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
 
 public class InvalidPublicationException extends ApiGatewayException {
 
     public static final String ERROR_MESSAGE_TEMPLATE =
-        "The Publication cannot be published because the following fields are not populated: %s";
+        "The Publication cannot be published because the following fields are not populated: ";
 
     public InvalidPublicationException(List<String> missingFields) {
-        super(String.format(ERROR_MESSAGE_TEMPLATE, String.join(", ", missingFields)));
+        super(ERROR_MESSAGE_TEMPLATE + String.join(", ", missingFields));
```
Comment (axthosarouris) [inline publication-commons/src/main/java/no/unit/nva/publication/exception/InvalidPublicationException.java:13]: simpler expression makes it easier to test the contents

File: build.gradle [hunk 4]
```diff
@@ -4,6 +4,7 @@ plugins {
     id 'checkstyle'
     id 'pmd'
     id 'base'
+    id "io.freefair.lombok" version "5.3.0"
```
Comment (janbj) [inline build.gradle:7]: What are we using lombok for?

**Review by janbj — APPROVED** [review]

**Review by brinxmat — APPROVED** [review]

**Pull request merged**